flipperdevices/flipperzero-firmware
# [FL-939] Nfc detection by CLI command

# What's new

- Add CLI command nfc_detect to start NFC detection
- After command launched, if NFC is not busy, start NFC detection with limited time and print result

# Verification

- Build for f4 target
- Build for f5 target
- Verify that command starts only when NFC is not busy.
- Test without NFC card. Command must exit in ~2 s
- Test with NFC card. Command must exist as fast as card is found, printing info.

# Checklist (do not modify)
